app/controllers/cart_items_controller.rb

```diff
@@ -17,12 +17,17 @@ def create
 
   def update
     item = Item.find(params[:id].to_i)
-    @cart.add_item(item.id)
-    redirect_to cart_items_path
+    if params[:edit_action] == 'add'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.